### PR TITLE
docs(reference): update Bitbucket api reference url to use 2.0

### PR DESCRIPTION
**Pull request by dorbin**
#1910

### CODE REVIEW

File: reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The pipeline UI exposes the following fields for the Bitbucket file artifact:
     </tr>
     <tr>
       <td><strong>File path</strong></td>
-      <td>The full path (including filename) for retrieval via the Bitbucket API. Example: <code>https://api.bitbucket.org/1.0/repositories/$ORG/$REPO/raw/$VERSION/$FILEPATH</code>.</td>
+      <td>The full path (including filename) for retrieval via the Bitbucket API. Example: <code>https://api.bitbucket.org/2.0/repositories/$ORG/$REPO/raw/$VERSION/$FILEPATH</code>.</td>
```
Comment (plumpy) [inline reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md]: You also need to change `raw` to `src` in all these URLs...

**Review by plumpy — APPROVED** [review]

File: reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md (hunk 1 above)
Comment (dorbin) [inline reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md]:
```suggestion
      <td>The full path (including filename) for retrieval via the Bitbucket API. Example: <code>https://api.bitbucket.org/2.0/repositories/$ORG/$REPO/src/$VERSION/$FILEPATH</code>.</td>
```

File: reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ The following are the fields that make up a Bitbucket file artifact:
 | Field | Explanation |
 |-|-----------|
 | `type` | Always `bitbucket/file`. |
-| `reference` | The full path (including filename) for retrieval via the Bitbucket API. Example: `https://api.bitbucket.org/1.0/repositories/$ORG/$REPO/raw/$VERSION/$FILEPATH`. |
+| `reference` | The full path (including filename) for retrieval via the Bitbucket API. Example: `https://api.bitbucket.org/2.0/repositories/$ORG/$REPO/raw/$VERSION/$FILEPATH`. |
```
Comment (dorbin) [inline reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md]:
```suggestion
| `reference` | The full path (including filename) for retrieval via the Bitbucket API. Example: `https://api.bitbucket.org/2.0/repositories/$ORG/$REPO/src/$VERSION/$FILEPATH`. |
```

File: reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ as it would appear in a pipeline definition:
 ```json
 {
   "type": "bitbucket/file",
-  "reference": "https://api.bitbucket.org/1.0/repositories/org/repo/raw/master/manifests/config.yaml",
+  "reference": "https://api.bitbucket.org/2.0/repositories/org/repo/raw/master/manifests/config.yaml",
```
Comment (dorbin) [inline reference/artifacts-with-artifactsrewrite/types/bitbucket-file/index.md]:
```suggestion
  "reference": "https://api.bitbucket.org/2.0/repositories/org/repo/src/master/manifests/config.yaml",
```

File: reference/artifacts/types/bitbucket-file/index.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ stage.
 | Field | Explanation |
 |-|-----------|
 | `type` | Always `bitbucket/file`. |
-| `reference` |  The full path including filename for retrieval via the Bitbucket API. `https://api.bitbucket.org/1.0/repositories/$ORG/$REPO/raw/$VERSION/$FILEPATH`. For more info, see the documentation [here](https://confluence.atlassian.com/bitbucket/src-resources-296095214.html).
+| `reference` |  The full path including filename for retrieval via the Bitbucket API. `https://api.bitbucket.org/2.0/repositories/$ORG/$REPO/raw/$VERSION/$FILEPATH`. For more info, see the documentation [here](https://confluence.atlassian.com/bitbucket/src-resources-296095214.html).
```
Comment (dorbin) [inline reference/artifacts/types/bitbucket-file/index.md]:
```suggestion
| `reference` |  The full path including filename for retrieval via the Bitbucket API. `https://api.bitbucket.org/2.0/repositories/$ORG/$REPO/src/$VERSION/$FILEPATH`. For more info, see the documentation [here](https://confluence.atlassian.com/bitbucket/src-resources-296095214.html).
```

File: reference/artifacts/types/bitbucket-file/index.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ stage.
 ```json
 {
   "type": "bitbucket/file",
-  "reference": "https://api.bitbucket.org/1.0/repositories/org/repo/raw/master/manifests/config.yaml",
+  "reference": "https://api.bitbucket.org/2.0/repositories/org/repo/raw/master/manifests/config.yaml",
```
Comment (dorbin) [inline reference/artifacts/types/bitbucket-file/index.md]:
```suggestion
  "reference": "https://api.bitbucket.org/2.0/repositories/org/repo/src/master/manifests/config.yaml",
```

**Review by plumpy — APPROVED** [review]
A++++++ :dango: :dango: :dango: 

Thanks!

**Pull request merged**